lib/version.js

```diff
@@ -0,0 +1 @@
+export const version = require('../package.json').version;
```

Comment:
Is this going to work in the created/exported package? I.e. does it always include the `../package.json` file?

Reply:
It should - we are doing it already for another package (bqplot-image-gl).